### PR TITLE
Complete readme

### PR DESCRIPTION
**Pull request by M5oul**
### Add
- how to contribute.
- repository content.
- how this part of YunoHost works and interface with other YunoHost components.
- dependencies.

### CODE REVIEW

**Comment by Psycojoker** [comment]
Maybe add a reference to the action map? Otherwise looks good to me :)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
-Please report issues here (no registration needed):
-https://dev.yunohost.org/projects/yunohost/issues
+# YunoHost core
+## Issues
+- [Please report issues on YunoHost bugtracker](https://dev.yunohost.org/projects/yunohost/issues) (no registration needed).
+
+## Contribute
+- You could develop on this repository using [ynh-dev tool](https://github.com/YunoHost/ynh-dev) with `use-git`  sub-command.
```
Comment (julienmalik) [inline README.md]: You can develop

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+- On this repository we are working [following this workflow](https://yunohost.org/#/build_system_en): `stable <— testing <— branch`.
```
Comment (julienmalik) [inline README.md]: 'On this repository we are following this workflow' sounds better

File: README.md [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
+## Repository content
+- Shell [application helpers](https://yunohost.org/#/packaging_apps_helpers_en).
+- Services configuration templates.
+- Modules for the XMPP server Metronome.
+- Debian files for package creation.
+- Locales for translations of `yunohost` command.
+- YunoHost core Python 2.7 scripts.
+
+## How does it works?
+- Python core scripts are accessible through two interfaces thanks to [moulinette framework](https://github.com/YunoHost/moulinette):
```
Comment (julienmalik) [inline README.md]: thanks to the moulinette framework

File: README.md [hunk 4]
```diff
@@ -1,2 +1,27 @@
...
+ - [CLI](https://en.wikipedia.org/wiki/Command-line_interface) for `yunohost` command.
+ - [API](https://en.wikipedia.org/wiki/Application_programming_interface) for [web administration module](https://github.com/YunoHost/yunohost-admin) (other modules could be implemented).
+- You could found more details about how YunoHost works on this [documentation (in french)](https://yunohost.org/#/package_list_fr).
```
Comment (julienmalik) [inline README.md]: You can find

**Comment by Psycojoker** [comment]
Re-reading, I realise that we are missing:

* link to yunohost.org explaining what this repository is about
* licence

Otherwise looks good :)

**Comment by M5oul** [comment]
It's ok for me.
If you are agree, we could merge this one as there isn't any conflicts nor testing possible issues (yep, it's doc 🐱 ).

**Comment by Psycojoker** [comment]
Thx :)

**Pull request merged**